boards/st/nucleo_h723zg/nucleo_h723zg.dts

```diff
@@ -124,7 +126,7 @@
 
 &rtc {
 	clocks = <&rcc STM32_CLOCK_BUS_APB4 0x00010000>,
-		 <&rcc STM32_SRC_LSI RTC_SEL(2)>;
+		 <&rcc STM32_SRC_LSE RTC_SEL(1)>;
```

Comment:
Either split in a dedicated commit or add a line in commit message to explain that you change clock for xxx reason.
Not that splitting is preferred in general as it allows to easily revert in case of trouble.